perftest/omrperftest.mk

```diff
@@ -39,7 +39,7 @@ endif
 all: test
 	
 omr_perfgctest:
-	./omrgctest -configListFile=perftest/gctest/configuration/perfConfigListFile.txt -keepVerboseLog
+	./omrgctest --gtest_filter="perfTest*" -keepVerboseLog
 	./omrperfgctest
```

Comment:
the perf test was only running the tests in perfConfigListFile, that's why it filters the functional tests. @renfeiw what do you think?